vector/src/main/res/values/strings.xml

```diff
@@ -3653,4 +3653,29 @@
         <item quantity="one">At least %1$s option is required</item>
         <item quantity="other">At least %1$s options are required</item>
     </plurals>
+    <plurals name="poll_option_vote_count">
+        <item quantity="one">%1$s vote</item>
+        <item quantity="other">%1$s votes</item>
+    </plurals>
+    <plurals name="poll_total_vote_count_before_ended_and_voted">
+        <item quantity="one">Based on %1$s vote</item>
+        <item quantity="other">Based on %1$s votes</item>
+    </plurals>
+    <plurals name="poll_total_vote_count_before_ended_and_not_voted">
+        <item quantity="zero">No votes cast</item>
+        <item quantity="one">%1$s vote cast. Vote to the see the results</item>
+        <item quantity="other">%1$s votes cast. Vote to the see the results</item>
+    </plurals>
+    <plurals name="poll_total_vote_count_after_ended">
+        <item quantity="one">Final result based on %1$s vote</item>
+        <item quantity="other">Final result based on %1$s votes</item>
+    </plurals>
+    <string name="poll_end_action">End poll</string>
+    <string name="a11y_poll_winner_option">winner option</string>
+    <string name="end_poll_confirmation_title">End poll</string>
+    <string name="end_poll_confirmation_description">Are you sure you want to end this poll? This will stop people from being able to vote and will display the final results of the poll.</string>
```

Comment:
Saying "Are you sure" is bad for the user experience (too stressful).
The dialog title should be:
> End this poll?

And the content:
> Ending the poll will stop people from being able to vote and will display the final results of the poll.

Reply:
I agree. We may consider checking other places we have this. Done.